cloudwatch/cloudwatch.go

```
@@ -545,6 +556,9 @@ func (output *OutputPlugin) setGroupStreamNames(e *Event) {
 }
 
 func (output *OutputPlugin) createStream(e *Event) (*logStream, error) {
```

Would like to double confirm on the `autoCreateStream` check. Does this mean `createStream` will only be called when `autoCreateStream` enable? the error logs is a bit wried to me since it does not check log stream existing or not at this place so kind of make this function coupling with `getLogStream` function too tight.

---

`createStream` will always be called but it will fail fast if autoCreateStream is disabled and `output.client.CreateLogStream` won't be called.

Which bit looks weird? I'd be happy to sync to clarify anything